python313Packages.app-model: 0.4.0 -> 0.5.0

fabaff opened the pull request:
Diff: https://github.com/pyapp-kit/app-model/compare/v0.4.0...v0.5.0

Changelog: https://github.com/pyapp-kit/app-model/blob/v0.5.0/CHANGELOG.md


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff committed:
python313Packages.app-model: 0.4.0 -> 0.5.0

Diff: https://github.com/pyapp-kit/app-model/compare/v0.4.0...v0.5.0

Changelog: https://github.com/pyapp-kit/app-model/blob/v0.5.0/CHANGELOG.md

wolfgangwalther commented:
> The diff for the temporary merge commit [25472ab](https://github.com/NixOS/nixpkgs/commit/25472ab7bc1b8e34d5585809ff3159a431ae110f) is empty. The changes in this PR have almost certainly already been merged to the target branch.

That's clearly wrong, reverting the auto-close feature in https://github.com/NixOS/nixpkgs/pull/462820.

fabaff committed:
python313Packages.app-model: remove disabled

fabaff committed:
python313Packages.psygnal: 0.14.0 -> 0.15.0

Diff: https://github.com/pyapp-kit/psygnal/compare/v0.14.0...v0.15.0

Changelog: https://github.com/pyapp-kit/psygnal/blob/v0.15.0/CHANGELOG.md

fabaff committed:
python313Packages.psygnal: remove disabled

fabaff committed:
napari: 0.6.4 -> 0.6.6

Changelog: https://github.com/napari/napari/releases/tag/v0.6.6

fabaff commented:
Depends on #465254
